test/features/member_test.exs

```diff
@@ -31,7 +31,7 @@ defmodule Community.Feature.MemberTest do
     assert_delivered_email Community.Email.admin_member_added(last_member)
   end
 
-  test "contacts a member that is available for hire" do
+  test "contacts a member that is available for hire", %{organization: _organization} do
```

Comment:
Do you have to use these? I think you can ignore these options when you don't need them.

Reply:
Nope. I'll remove them. This was more of me remembering that it's available to use in feature and controller tests.